melodic/distribution.yaml

```diff
@@ -6084,6 +6084,33 @@ repositories:
       url: https://github.com/ros-gbp/pr2_mechanism_msgs-release.git
       version: 1.8.2-0
     status: unmaintained
+  pr2_navigation:
+    doc:
+      type: git
+      url: https://github.com/PR2/pr2_navigation.git
```

Comment:
In Kinetic, the source and doc repository is in the PR2-prime organization: https://github.com/ros/rosdistro/blob/82d59b736154de4f465e9bfebf4d0ac662a52cd2/kinetic/distribution.yaml#L10033 .  Why the change?

Reply:
We had to release from pr2-prime because we do not have access to pr2 repository at that time, now we became admin of pr2 organization, so we are now ready to release them from pr2 repository , not pr2-prime.
Cc: @davefeilseifer